assets/src/blocks/TableOfContents/getTableOfContentsStyle.js

```diff
@@ -0,0 +1,17 @@
+import {getStyleFromClassName} from '../../functions/getStyleFromClassName';
+
+// Map for old attribute 'table_of_contents_style'
```

Comment:
Same comment as earlier, should remain `submenu_style`

Reply:
Change reverted.